profiles/package.use.mask

```diff
@@ -22,3 +22,9 @@ app-editors/leksah codemirror
 <=dev-haskell/gnuidn-0.2.2 test
 <=dev-haskell/system-filepath-0.4.14 test
 <=dev-haskell/dbus-0.10.13 test
+
+# hololeap <hololeap@gmail.com> (22 Jan 2021)
+# Building these with ghc-lib is currently untested
+dev-haskell/ghcide ghc-lib
+dev-haskell/hie-compat ghc-lib
+dev-haskell/hls-hlint-plugin ghc-lib
```

Comment:
Minor point: we generally top-post new entries to these files (like package.mask).

Reply:
I turned it into three separate commits.

Edit: @ezzieyguywuf explained that this means putting newest entries at the top. It's been turned back into one commit and this has been fixed.